app/views/course/assessment/assessments/_form.html.slim

```diff
@@ -14,6 +14,8 @@
     = f.input :draft
   - unless @assessment.autograded?
     = f.input :tabbed_view
+  - if @assessment.autograded?
+    = f.input :skippable, label: t('.skippable')
```

Comment:
Do we need to handle the case where `!@assessment.autograded?` - ie. `skippable` is `true`?